lessons/r/ggplot2/Lindsayggplot2.Rmd

```diff
@@ -0,0 +1,152 @@
+---
+always_allow_html: yes
+layout: page
+title: 'Plotting in R: ggplot2'
+visible: false
+tags:
+  - r
+  - intermediate
+output:
+  md_document:
+    variant: markdown_github
+---
+
+# INTRODUCTION TO GGPLOT2
+* ##### Author: Lindsay Coome
+* ##### Lesson Topic: Plotting in R using ggplot2
```

Comment:
Also include the link to the github repo material (https://github.com/UofTCoders/studyGroup/tree/gh-pages/lessons/r/ggplot2)